shared/syncer.js

```diff
@@ -24,7 +24,8 @@ if (isServer) {
   var serverOnly_qs = 'qs';
   var qs = require(serverOnly_qs);
 } else {
-  Backbone.$ = window.$ || require('jquery');
+  var $ = window.$ || require('jquery');
```

Comment:
jshint was complaining about $ not being on line 138. (`$.getJSON(url, this.params, function(resp) {"`)
I sax two options to fix this : the one above or using `window.$.getJSON`

Which one do you prefer ?